op-node/p2p/pings.go

```diff
@@ -0,0 +1,134 @@
+package p2p
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+	"golang.org/x/time/rate"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-service/clock"
+)
+
+const (
+	pingRound                 = 10 * time.Minute
```

Comment:
A single ping per host per 10min is going to have very high variability. An hour of peer-time would only give you 6 data points to construct an average from. And if any of those calls are out of the norm, it'll severely affect the average for a long while.

I suggest lowering this considerably, somewhere between 1 and 5 minutes. I also have a comment further down about pinging multiple times which would help this.

Reply:
Will lower it to 3 minutes.